samples/ResourceGroup/copy-resourcegroup-tag/README.md

```diff
@@ -26,7 +26,7 @@ $assignment
 
 ````cli
 # Create the Policy Definition
-az policy definition create --name 'copy-resourcegroup-tag' --display-name 'Copy resource group tag to resource' --description 'Copy a specific tag from the resource group to the resource.  For example, a tag named example on the resource group will be copied to the resources.  Change the word example in the policy rule to change the name of the tag copied to a resource.' --rules 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/ResourceGroup/copy-resourcegroup-tag/azurepolicy.rules.json' --mode Indexed
+az policy definition create --name 'copy-resourcegroup-tag' --display-name 'Copy resource group tag to resource' --description 'Copy a tag specified in the parameter value from the resource group to the resource.  For example, a tag named costCenter on the resource group will be copied to the resources. Provide the actual tag name in tagName parameter at time of policy assignment.' --rules 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/ResourceGroup/copy-resourcegroup-tag/azurepolicy.rules.json' --mode Indexed
 
 # Create the Policy Assignment
 az policy assignment create --name <assignmentname> --scope <scope> --policy "copy-resourcegroup-tag" 
```

Comment:
az cli example should include parameter:

--params "{'tagName':{'value': 'costCenter'}}"